scene/3d/camera_3d.h

```diff
@@ -72,8 +67,8 @@ class Camera3D : public Node3D {
 	real_t fov = 75.0;
 	real_t size = 1.0;
 	Vector2 frustum_offset;
-	real_t near = 0.05;
-	real_t far = 4000.0;
+	real_t camera_near = 0.05;
+	real_t camera_far = 4000.0;
```

Comment:
I'd suggest something like this since adding "camera_" as prefix doesn't really add meaningful information. And we have some classes where private members use `_` as a prefix, though it's not a convention we use explicitly.
```suggestion
	// _ prefix to avoid conflict with Windows defines.
	real_t _near = 0.05;
	real_t _far = 4000.0;
```

Alternatively, `near_distance`/`far_distance` if we want to be explicit at the cost of verbosity.

Reply:
I added the _ prefix